packages/appstore/src/components/transaction-list/completed-transaction/completed-transaction.tsx

```diff
@@ -126,11 +127,14 @@ const NonPendingTransaction = ({ transaction }: TNonPendingTransaction) => {
                     weight='lighter'
                     line_height={is_mobile ? 'm' : 's'}
                 >
-                    Balance: {formatAmount(balance_after)} {account_currency_config?.code}
+                    {localize('Balance: {{amount}} {{currency_code}}', {
```

Comment:
Dont think this will work, should use Localize component instead

Reply:
Done